### PR TITLE
Tradeoffs in the persistent congestion threshold

### LINKED ISSUE #4195: Recovery-31: Limits on kPersistentCongestionThreshold values?
Section 7.6.1: 

	The RECOMMENDED value for kPersistentCongestionThreshold is 3, which is approximately equivalent to two TLPs before an RTO in TCP.

Any values that needs to be explicitly forbidden? For example does it need a sentence saying that it MUST be larger than X? I would think that 1 would be to low and allow slow start after just one RTT+variations? I understand that likely would give the application bad behavior but likely also have negative impact on other applications if the QUIC stack ends up doing slow start crashing into the wall and then slow starting again?

### PR DESCRIPTION
**Pull request by janaiyengar**
Closes #4195.

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -951,8 +951,15 @@ This duration allows a sender to send as many packets before establishing
 persistent congestion, including some in response to PTO expiration, as TCP does
 with Tail Loss Probes ({{RACK}}) and a Retransmission Timeout ({{?RFC5681}}).
 
-The RECOMMENDED value for kPersistentCongestionThreshold is 3, which is
-approximately equivalent to two TLPs before an RTO in TCP.
+As the value kPersistentCongestionThreshold gets larger, the sender becomes
+increasingly unresponsive to persistent congestion in the network, which can
+result in aggressive sending into a congested network. Too small a value can
+result in a sender declaring persistent congestion unnecessarily, resulting in
+reduced throughput for the sender.
```
Comment (ianswett) [inline draft-ietf-quic-recovery.md:958]:
```suggestion
Larger values of kPersistentCongestionThreshold cause the sender to
become less responsive to persistent congestion in the network, which can
result in aggressive sending into a congested network. Too small a value can
result in a sender declaring persistent congestion unnecessarily, resulting in
reduced throughput for the sender.
```

**Review by ianswett — APPROVED** [review]
A suggestion

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (gloinul) [reply draft-ietf-quic-recovery.md:958]: Like @ianswett proposal better.

**Review by gloinul — COMMENTED** [review]
Thinks this resolve my issue, but would prefer Ian's proposal.

**Review by ianswett — APPROVED** [review]

**Pull request merged**